packages/ringcentral-integration/modules/Analytics/index.js

```diff
@@ -405,11 +410,35 @@ export default class Analytics extends RcModule {
     }
   }
 
+  _conferenceControlClickParticipantsArea(action) {
```

Comment:
remove the router criteria inside the  `_simplifiedCallControlClickHangup `&`_conferenceControlClickParticipantsArea `, replace it with action dispatcher